docs/content/preview/develop/best-practices-ycql.md

```diff
@@ -64,6 +64,10 @@ TTL does not apply to transactional tables and so, its unsupported in that conte
 
 Use YugabyteDB-specific [client drivers](../../drivers-orms/) because they are cluster- and partition-aware, and support `jsonb` columns.
 
+## Make sure the application uses new nodes
```

Comment:
This is not applicable to YCQL drivers. The upstream drivers on which our YCQL drivers are based on, regularly check for new nodes in the cluster and establish connections with them.